src/test/groovy/org/gradle/android/SimpleAndroidApp.groovy

```diff
@@ -12,15 +12,19 @@ class SimpleAndroidApp {
     private final boolean dataBindingEnabled
     private final boolean kotlinEnabled
     private final boolean kaptWorkersEnabled
+    private final boolean roomExtensionConfigured
+    private final boolean roomAnnotationProcessorArgumentEnabled
 
-    private SimpleAndroidApp(File projectDir, File cacheDir, VersionNumber androidVersion, VersionNumber kotlinVersion, boolean dataBindingEnabled, boolean kotlinEnabled, boolean kaptWorkersEnabled) {
+    private SimpleAndroidApp(File projectDir, File cacheDir, VersionNumber androidVersion, VersionNumber kotlinVersion, boolean dataBindingEnabled, boolean kotlinEnabled, boolean kaptWorkersEnabled, boolean roomExtensionConfigured, boolean roomAnnotationProcessorArgumentEnabled) {
         this.dataBindingEnabled = dataBindingEnabled
         this.projectDir = projectDir
         this.cacheDir = cacheDir
         this.androidVersion = androidVersion
         this.kotlinVersion = kotlinVersion
         this.kotlinEnabled = kotlinEnabled
         this.kaptWorkersEnabled = kaptWorkersEnabled
+        this.roomExtensionConfigured = roomExtensionConfigured
+        this.roomAnnotationProcessorArgumentEnabled = roomAnnotationProcessorArgumentEnabled
```

Comment:
Just trying to be clear - `withRoomExtensionEnabled()` feels odd because what does mean to "enable" it?  That is is "configured" seemed more specific and clear what we are doing IMO.

Reply:
Ah makes sense.